__tests__/patch-state.spec.ts

```diff
@@ -46,48 +46,69 @@ describe('view state & patch state', () => {
       ],
     ]);
 
-    test(`View state${process.env.NEAR_RUNNER_NETWORK !== 'testnet' ? '' : '(skipping on testnet)'}`, async () => {
+    test(`View state${
+      process.env.NEAR_RUNNER_NETWORK !== "testnet"
+        ? ""
+        : "(skipping on testnet)"
+    }`, async () => {
       await runner.runSandbox(async ({ contract, ali }) => {
-        await ali.call(contract, "set_status", { message: "hello" })
+        await ali.call(contract, "set_status", { message: "hello" });
 
         const state = await contract.viewState();
 
         // Get raw value
         let data = state.get_raw("STATE");
 
         // deserialize from borsh
-        const statusMessage: StatusMessage = borsh.deserialize(schema, StatusMessage, data);
+        const statusMessage: StatusMessage = borsh.deserialize(
+          schema,
+          StatusMessage,
+          data
+        );
 
         expect(statusMessage.records[0]).toStrictEqual(
-          new Record({ k: ali.accountId, v: 'hello' })
-        )
+          new Record({ k: ali.accountId, v: "hello" })
+        );
       });
     });
 
-
-    test(`Patch state${process.env.NEAR_RUNNER_NETWORK !== 'testnet' ? '' : ' (skipping on testnet)'}`, async () => {
+    test(`Patch state${
+      process.env.NEAR_RUNNER_NETWORK !== "testnet"
+        ? ""
+        : " (skipping on testnet)"
+    }`, async () => {
```

Comment:
Test-skipping now explained above

```suggestion
    test("Patch state", async () => {
```